frontend/src/scenes/pipeline/AppsManagement.tsx

```diff
@@ -0,0 +1,394 @@
+import { LemonBanner, LemonDivider, LemonLabel, LemonTable, Tooltip } from '@posthog/lemon-ui'
+import { Popconfirm } from 'antd'
+import { useActions, useValues } from 'kea'
+import { IconDelete, IconLock, IconLockOpen } from 'lib/lemon-ui/icons'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { LemonInput } from 'lib/lemon-ui/LemonInput'
+import { Link } from 'lib/lemon-ui/Link'
+import { preflightLogic } from 'scenes/PreflightCheck/preflightLogic'
+import { SceneExport } from 'scenes/sceneTypes'
+
+import { PluginInstallationType, PluginType } from '~/types'
+
+import { appsManagementLogic } from './appsManagementLogic'
+import { RenderApp } from './utils'
+
+export const scene: SceneExport = {
+    component: AppsManagement,
+    logic: appsManagementLogic,
+}
+
+export function AppsManagement(): JSX.Element {
+    const { canInstallPlugins, canGloballyManagePlugins } = useValues(appsManagementLogic)
+    const { globalPlugins, localPlugins } = useValues(appsManagementLogic)
+
+    if (!canInstallPlugins || !canGloballyManagePlugins) {
+        return <>You don't have permission to manage apps.</>
+    }
+
+    return (
+        <div className="pipeline-apps-management-scene">
+            <OutOfSyncApps />
+            <InstallFromUrl />
+            <InstallLocalApp />
+            <InstallSourceApp />
```

Comment:
This page has too much content for a structure this flat. It would be more readable if there were `h2` for the sections present: "Available apps" (installing new ones) and "Installed apps" (managing the ones installed already)